web/themes/new_weather_theme/templates/partials/daily-forecast-list-item.html.twig

```diff
@@ -20,24 +20,27 @@ only be a nighttime period.
         <span class="display-block">{{ periods[0].monthAndDay }}</span>
       </time>
 
-      {% for period in periods %}
-        {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'period': period } %}
-      {% endfor %}
+      {% include '@new_weather_theme/partials/daily-high-low.html.twig' with { 'periods': periods } %}
     </div>
     <div class="grid-row grid-gap">
-      {% for period in periods %}
-        {% if period.isDaytime %}
-          {% set label = "Day" | t %}
-          {% set lineColor = "yellow" %}
-          {% set marginUnit = 3 %}
-        {% else %}
-          {% set label = "Night" | t %}
-          {% set lineColor = "indigo" %}
-          {% set marginUnit = 1 %}
-        {% endif %}
-        <div class="daily-forecast-summary-area grid-col-12 tablet:grid-col-6 margin-bottom-{{ marginUnit }} tablet:margin-bottom-1">
+        {% for period in periods %}
+            {% set marginUnit = 1 %}
+            {% set tabletCol = 6 %}
+            {%  if period.isOvernight %}
+                {% set label = "Overnight" | t %}
+                {% set lineColor = "indigo" %}
+                {% set tabletCol = 12 %}
+            {% elseif period.isDaytime %}
+                {% set label = "Day" | t %}
+                {% set lineColor = "yellow" %}
+                {% set marginUnit = 3 %}
+            {% else %}
+                {% set label = "Night" | t %}
+                {% set lineColor = "indigo" %}
```

Comment:
With three periods the first two both need a 3 units of margin-bottom, so I am thinking this should change to
```suggestion
        {% for period in periods %}
            {% set marginUnit = 3 %}
            {% set tabletCol = 6 %}
            {%  if period.isOvernight %}
                {% set label = "Overnight" | t %}
                {% set lineColor = "indigo" %}
                {% set tabletCol = 12 %}
            {% elseif period.isDaytime %}
                {% set label = "Day" | t %}
                {% set lineColor = "yellow" %}               
            {% else %}
                {% set label = "Night" | t %}
                {% set lineColor = "indigo" %}
                 {% set marginUnit = 1 %}
```